src/commands/compile.civet

```diff
@@ -0,0 +1,17 @@
+{ SlashCommandBuilder, ChatInputCommandInteraction, CacheType, version as djsVersion } from discord.js
+compile, { format } from ../compile.civet
+
+data := new SlashCommandBuilder()
+		.setName "compile"
+		.setDescription "Compiles some Civet."

```

Comment:
Can you add mention in the description that this is private, unlike ` ```civet``` ` messages in the channel?